lib/ExpandablePicker/ExpandablePicker.less

```diff
@@ -38,7 +38,7 @@
 
 /* Current Value */
 .moon-expandable-picker-current-value {
-	.moon-body-text;
+	.moon-item-body-text;
```

Comment:
With this change, for 1080p, the `line-height` will change from `33px` to `36px` in a non-latin locale, and will change from `39px` to `39.5999984741211px` in other locales. I wanted to verify that this was intentional, thanks!